src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,205 @@
+                insertDriverStatement.setLong(1, driver.getId());
+                insertDriverStatement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Couldn't insert driver for the car" + car, e);
+        }
+
+    }
+
+    @Override
+    public Car get(Long id) {
+        Car car = null;
+        String getCarByIdRequest = "SELECT c.id, c.model, c.manufacturer_id, m.name, m.country \n"
+                + "FROM cars c JOIN manufacturers m\n"
+                + "ON c.manufacturer_id = m.id\n"
+                + "WHERE c.id = ? AND c.is_deleted = FALSE;";
+        try (Connection connection =
+                     ConnectionUtil.getConnection();
+                PreparedStatement getCarStatement =
+                        connection.prepareStatement(getCarByIdRequest)) {
+            getCarStatement.setLong(1, id);
+            ResultSet resultSet = getCarStatement.executeQuery();
+            if (resultSet.next()) {
+                car = parseCarFromResultSetWithManufacturer(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get car from DB with id " + id, e);
+        }
+        if (car != null) {
+            car.setDrivers(getDriversForCar(id));
+        }
+        return car;
+    }
+
+    private Car parseCarFromResultSetWithManufacturer(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setId(resultSet.getObject("id", Long.class));
+        car.setModel(resultSet.getString("model"));
+        Manufacturer manufacturer = new Manufacturer(
+                resultSet.getObject("manufacturer_id", Long.class),
+                resultSet.getString("name"),
+                resultSet.getString("country"));
+        car.setManufacturer(manufacturer);
+        return car;
+    }
+
+    private List<Driver> getDriversForCar(Long id) {
+        String getDriversForCarRequest =
+                "SELECT id, name, licenseNumber FROM drivers d"
+                        + " JOIN cars_drivers cd ON d.id = cd.driver_id"
+                        + " WHERE cd.car_id = ? AND is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getDriversStatement =
+                        connection.prepareStatement(getDriversForCarRequest)) {
+            getDriversStatement.setLong(1, id);
+            ResultSet resultSet = getDriversStatement.executeQuery();
+            List<Driver> driversList = new ArrayList<>();
+            while (resultSet.next()) {
+                driversList.add(parseDriverFromRS(resultSet));
+            }
+            return driversList;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get drivers for car id + " + id, e);
+        }
+    }
+
+    private Driver parseDriverFromRS(ResultSet resultSet) throws SQLException {
```

Comment:
```suggestion
    private Driver parseDriver(ResultSet resultSet) throws SQLException {
```
'FromRS' is not needed since method have ResultSet as parameter and obviously it would parse Driver from the parameter